azext_edge/edge/providers/orchestration/common.py

```diff
@@ -59,3 +59,29 @@ class KubernetesDistroType(Enum):
 
 class IdentityUsageType(Enum):
     dataflow = "dataflow"
+
+
+class SchemaType(Enum):
+    """value is user friendly, full_value is the service friendly"""
```

Comment:
nit - is the service friendly? debatable

Reply:
very debatable